src/managers/amazon-manager/amazon-manager.ts

```diff
@@ -0,0 +1,183 @@
+import {DOMParser} from 'linkedom/worker';
+import {getActiveTab, getHtml} from "../../utils/utils";
+import {FullTransaction, OrderInfo, Transaction} from "./types";
+import * as statusAnd from "../../utils/statusAnd"
+
+export class AmazonManager {
```

Comment:
It better sets up adding new features to the project since you can construct the class.  It also helps with naming, since there's a clear difference of where the classes's method is coming from vs a directly imported function.

Reply:
> It better sets up adding new features to the project since you can construct the class

I don't know what this means, can you elaborate?

> It also helps with naming

The functions should be named pretty clearly, and the importing code can do something like `import * as ...` if it wants to namespace.

I should have written in the dev guide that I generally try to follow https://google.github.io/styleguide/tsguide.html#container-classes, sorry!